flytekit/configuration/__init__.py

```diff
@@ -227,11 +227,13 @@ def look_up_image_info(name: str, tag: str, optional_tag: bool = False) -> Image
 @dataclass(init=True, repr=True, eq=True, frozen=True)
 class ImageConfig(object):
     """
+    We recommend you to use ImageConfig.auto(img_name=None) to create an ImageConfig.
+    For example, ImageConfig.auto(img_name=""ghcr.io/flyteorg/flytecookbook:v1.0.0"") will create an ImageConfig.
+
     ImageConfig holds available images which can be used at registration time. A default image can be specified
     along with optional additional images. Each image in the config must have a unique name.
 
     Attributes:
-        default_image (str): The default image to be used as a container for task serialization.
```

Comment:
why remove this line?

Reply:
> why remove this line?
because there's a maintainer guide me to do this !
https://flyte-org.slack.com/archives/CP2HDHKE1/p1689062874403619

Reply:
cc @wild-endeavor